src/amo/components/Search/index.js

```diff
@@ -60,49 +77,66 @@ export class SearchBase extends React.Component {
       LinkComponent,
       count,
       enableSearchSort,
+      errorHandler,
       filters,
-      hasSearchParams,
       loading,
-      page,
+      paginationQueryParams,
```

Comment:
This is a bit odd though when first looked at though, agreed. I'll add some comments to it.